express-crud/server.ts

```diff
@@ -0,0 +1,139 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib';
+
+type Grade = {
+  gradeId?: number;
+  name: string;
+  course: string;
+  score: number;
+};
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: { rejectUnauthorized: false },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+      select * from "grades";
+    `;
+    const result = await db.query(sql);
```

Comment:
The best way to use TypeScript with pg is:
`await db.query<Grade>(sql)`
This tells TS that result.rows will be a Grade[].